homestar-runtime/src/event_handler.rs

```diff
@@ -189,15 +214,20 @@ where
                      swarm_event.handle_event(&mut self).await;
 
                 }
-
             }
+
+            // Poll cache for expired entries
+            self.cache.run_pending_tasks().await;
```

Comment:
we have our spawn loop below, right? So this can go?

Reply:
Ah yeah, missed updating this in the IPFS feature off case. Fixed. 👌